components/brave_wallet/browser/test/BUILD.gn

```diff
@@ -41,7 +42,7 @@ source_set("brave_wallet_unit_tests") {
       "//brave/components/brave_wallet/browser:ethereum_permission_utils",
       "//brave/components/brave_wallet/common",
       "//brave/components/brave_wallet/common:mojom",
-      "//brave/components/brave_wallet/renderer/test:brave_wallet_response_unit_tests",
+      "//brave/components/brave_wallet/renderer/test:unit_tests",
```

Comment:
I think this should be moved after https://github.com/brave/brave-core/blob/master/test/BUILD.gn#L200 instead.

Reply:
Fixed and interactive rebase squashed it into d17f0bdbff03fe6e299ea43ee030540a4ee08eca